### ISSUE TITLE
Register Template not working with assets

### ISSUE THREAD

**Issue by Uysim** [camertron/mosaico-rails#7]
Register template not working with assets with 'edres' and 'img' folders. Because it always find assets in ```Mosaico.vendor_template_root```

**Comment by camertron** [comment]
Hey @Uysim, sounds like a bug, thanks for letting me know :) Without access to your code however, I'm going to have a hard time reproducing and fixing it. Could you push up a test repository I could take a look at?

**Comment by galacto** [comment]
@camertron  I tried this today 

1.  Create a directory  lib/templates/kosmo 
2. Create an HTML file  lib/templates/kosmo/template-kosmo.html.
3. To reproduce lets copy all files from `mosaico-rails/vendor/assets/mosaico-0.16.0/mosaico/templates/versafix-1/` to `lib/templates/kosmo` folder

4. Register temaplate in `config/initializers/mosaico.rb` 
` Mosaico.register_template('kosmo', Rails.root.join('lib/templates/kosmo'))`

![screenshot 2019-03-03 at 21 43 32](https://user-images.githubusercontent.com/6483087/53698037-90fc8800-3dfd-11e9-9187-f2201c111b03.png)

Could you give any pointers? Thanks for help.

**Comment by galacto** [comment]
I forgot to mention that I tried to run rake task 
`bundle exec rake mosaico:make_thumbs['template']`

But its not listed in rails -T list

**Comment by camertron** [comment]
Hey @galacto, apologies for not responding to this. I haven't forgotten, just busy :)

My hunch is this has something to do with adding those missing assets to the precompile list. If you have spare time I would welcome a pull request, otherwise I'll see if I can take a look this coming week.

**Comment by camertron** [comment]
Ok @Uysim and @galacto, I was able to sort out the problem and just published v1.1.1 of the gem. Please give it a try and let me know if it solves your issue.

**Comment by camertron** [comment]
@galacto oh also, even though the `make_thumbs` rake task isn't listed when running `rails -T list`, the task should still be available to you. One thing to keep in mind is you'll need to run the command using the name of the template you want to generate thumbnails for. So for the kosmo template you would run `bundle exec rake mosaico:make_thumbs['kosmo']`.

**Comment by galacto** [comment]
@camertron this started working now. So I followed same steps that I listed above. It generated thumbs for sidebar but one more problem I am seeing now is its only generating placeholder for 543X150.png in public/mosaico/placeholders folder. I am trying to debug why its not generating other placeholder images. any help is appreciated. 

One more thing this time I came across bower error I had to change my bower registry  to registry.bower.io 
see here - https://gist.github.com/sheerun/c04d856a7a368bad2896ff0c4958cb00

p.s. still looking into placeholder generations
